app/modules/topics/tests/actions.test.js

```diff
@@ -9,25 +9,35 @@ describe(`actions`, (): void => {
   describe(`add`, (): void => {
 
     it(`returns a topic ADD action, when parameters are valid`, (): void => {
+      const userId = 'testtest12';
       const title = 'Lorem ipsum';
       const description = 'Lorem ipsum dolor sit amet';
+      const rootContentItemId = 'abcdefghij';
       const expectedAction: t.AddAction = {
         type: t.ADD,
         payload: {
           id: '',
+          userId,
           title,
           description,
+          rootContentItemId,
         },
       };
-      const generatedAction: t.AddAction = ((actions.add(title, description): any): t.AddAction);
+      // eslint-disable-next-line
```

Comment:
Please specify the specific rule(s) that you want to disable instead of disabling them all; that might allow errors to go undetected.